stackslib/src/net/api/getblock_v3.rs

```diff
@@ -69,6 +63,27 @@ pub struct NakamotoBlockStream {
     pub rowid: i64,
 }
 
+impl RPCNakamotoBlockRequestHandler {
+    pub fn new() -> Self {
+        Self { block_id: None }
+    }
+
+    pub fn get_stream_by_node(
```

Comment:
I'm not sure what's gained by this?

Reply:
Definitely nothing relevant and  no gain in terms of readability. Removed it.